scripts/output_factory.py

```diff
@@ -13,6 +13,7 @@ def create(output_type, output, wordcloud_title=None, tfidf_reduce_obj=None, nam
 
     if output_type == 'report':
         filename_and_path = path.join('outputs', 'reports', name + '.txt')
+        print('')
```

Comment:
Allegedly all you need is:
```suggestion
        print()
```